README.md

```diff
@@ -0,0 +1,16 @@
+# Satellite Ansible Roles
+
+Ansible Roles to setup / configure Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or FAM.
+
+Note: The roles here are not officially supported by Red Hat.
+
+## Rules for contributing Roles to this project
+
+* The use cases supported by the roles should be beneficial for many users.
+* Input variables externalized in `vars` folder for easy reuse.
```

Comment:
By `vars` you mean within each role itself? If thats the case, they should go into `defaults` since roles is intended to be internal variables to the role and defaults external.